src/easyreflectometry/project.py

```diff
@@ -22,7 +22,7 @@
 class Project:
     def __init__(self):
         self._info = self._default_info()
-        self._path = Path(os.path.expanduser('~'))
+        self._root_path = Path(os.path.expanduser('~'))
```

Comment:
is this `root` or is this `HOME`? It points to the current user's home directory, not the system root.

Reply:
changed name to `_path_project_parent`